### PR TITLE
Added a function to query the sports in a city, and slight modifications

### PR DESCRIPTION
**Pull request by JackyChiu**
In this PR I added:

- 'findSports' a Cloud Code function that returns the array of sports available in the city
- Current solution on testing the server code that hasn't been deployed

I modified:

- Tabbing of tempData json's (They weren't consistent with 4 space tabs lol)
- package.json to use our new links and dependencies

### CODE REVIEW

File: cloud/main.js [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-            respones.error('No Cities found');
+        error: function(error) {
+            response.error(error);
+        }
+    });
+});
+
+Parse.Cloud.define('findSports', function(request, response) {
+    var query = new Parse.Query('Cities');
+    query.equalTo('name', request.params.name);
```
Comment (BenEmdon) [inline cloud/main.js]: So in the request you're sending the city name to retrieve the sports?

File: tempData/Montreal.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-    },
-    "tennis": {
-      "name": "Tennis",
-      "schedule": {}
-    },
-    "volleyball": {
-      "name": "Volleyball",
-      "schedule": {}
+    "name": "Montreal",
+    "sports": {
```
Comment (BenEmdon) [inline tempData/Montreal.json]: Maybe give the test fixtures different sports so we can know the difference in testing.

**Comment by BenEmdon** [comment]
Just two comments. Otherwise LGTM 👍

File: cloud/main.js (hunk 1 above)
Comment (JackyChiu) [inline cloud/main.js]: Yes, the parameter goes takes a key-value set eg. 
```
'name' : 'Ottawa'
```

File: tempData/Montreal.json (hunk 2 above)
Comment (JackyChiu) [inline tempData/Montreal.json]: Yes, we can do that to test it out on the database.

Comment (JackyChiu) [inline tempData/Montreal.json]: I can confirm that findSports works for each city. 
In the database, I changed the following:
```
 "name": "Ottawa",
    "sports": {
        "Test": {
            "name": "Foo",
            "schedule": {}
        },
// more sports
```

Then ran `heroku local` (Still looking for sports in Ottawa btw):
```
11:20:03 AM web.1 |  travelnplay running on port 5000.
11:20:03 AM web.1 |  Searching through sports collection returns:
11:20:03 AM web.1 |  [ 'Test', 'Basketball', 'Tennis', 'Volleyball' ]
```

**Pull request merged**